PYTHON_UPGRADE_GUIDE.MD

```diff
@@ -0,0 +1,36 @@
+# Python Upgrade Runbook
+
+So you wanna upgrade PyTorch to support a new Python? Follow these steps in order! They are adapted from previous CUDA upgrade processes.
+
+### 0.Before starting upgrade process. Check the package availability
+
+Package availability to validate before starting upgrade process :
+
+1) Python package is available via python ftp [here](https://www.python.org/ftp/python/) - This is minimal requirement for all wheel builds
+
+2) Python package is available via anaconda [here](https://anaconda.org/anaconda/python/files) - This is required for conda builds
```

Comment:
We are going to start before this is ready so this is not really a requirement?

Reply:
Agreed. Let me add a the statement about that conda build can be done independently